### DISCUSSION TITLE
Basalt generators for the nether world ores

### DISCUSSION THREAD

**Discussion by Trinchezito** [BG-Software-LLC/SuperiorSkyblock2#968] [Denied Feature Requests] [answered]
Is there any way possible to add upgrades for Basalt blocks generators so that players can mine and find more nether materials? Such as ancient debris, quartz ore, crimson nylium and other nether blocks? 

![image](https://user-images.githubusercontent.com/98068350/156685841-07dfc256-5ec2-4a34-9d2b-475b9368bad3.png)

**Comment by OmerBenGera** [comment, marked as answer]
This can be achieved using the upgrades system. Here is a tutorial about it and how to set up custom generator rates:
https://wiki.bg-software.com/superiorskyblock/overview/upgrades
